PyOpenWorm/simpleProperty.py

```diff
@@ -125,11 +177,20 @@ class _DatatypeValueProperty (DatatypePropertyMixin, _ValueProperty):
 
 
 class ObjectProperty (_ObjectPropertyMixin, RealSimpleProperty):
-    pass
+
```

Comment:
Introducing defined values to the result. Designed to see the result of a get as what's been "staged" in defined_values plus what's in the database.